front-page.php

```diff
@@ -0,0 +1,31 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+
+  <section id="player">
+    <?php // get_template_part('partials/player'); ?>
+  </section>
+
+  <section id="page-programacion" class="page-overlay">
+    <?php get_template_part('partials/programacion'); ?>
+  </section>
+
+<?php
+  $page_sobre = get_page_by_path('sobre');
+
+  if ($page_sobre != null) {
```

Comment:
the empty return value of `get_page_by_path` is null which is false as a condition. so the check can just be on the variable (99% sure of this)